test/behaviour/resolution/ResolutionSteps.java

```diff
@@ -95,13 +96,28 @@ public void reasoned_keyspace_answer_size_is(final int expectedCount) {
         }
     }
 
+    @Then("answers are consistent across {int} executions in reasoned keyspace")
+    public void reasoned_keyspace_answers_are_consistent_across_n_executions(final int executionCount) {
```

Comment:
why are we prefixing with "reasoned_keyspace"? is this our current style?

Reply:
Updated to answers_are_..._in_reasoned_keyspace